isis/tests/BundleObservationSolveSettingsTests.cpp

```diff
@@ -608,7 +608,7 @@ TEST(BundleObservationSolveSettings, GroupConstructorBadOverExisting) {
 
 TEST(BundleObservationSolveSettings, PositionStringToOptionBadValue)
 {
-  QString message = "Unknown bundle instrument position solve option foo";
+  QString message = "Unknown bundle instrument position solve option foo.";
```

Comment:
Did these change with the TestPreferences?

Reply:
These were reverted changes from #3820 to test that these tests are now passing with the added period since they are now using the TestPreferences file. My assumption is that TestPreferences will not be modified.